include/mrdox/Platform.hpp

```diff
@@ -53,16 +48,14 @@ namespace mrdox {
 
 // (unknown)
 #elif defined(__GNUC__)
-# if defined(MRDOX_TOOL) // building library
-#   define MRDOX_DECL
-# else
-#   define MRDOX_DECL __attribute__((visibility("default")))
-#endif
-# define MRDOX_VISIBLE __attribute__((visibility("default")))
+#  define MRDOX_DECL __attribute__((visibility("default")))
+#  define MRDOX_VISIBLE __attribute__((visibility("default")))
```

Comment:
are the plugins able to see the symbols in the executable?

Reply:
well the locally built version does.